govc/test/namespace.bats

```diff
@@ -32,7 +32,7 @@ load test_helper
     vcsim_env
 
     # need to set up some dependencies
-    govc cluster.create Workload-Cluster
+    govc cluster.create WCP-Cluster
```

Comment:
Do you think it's helpful to drop a comment here about the naming of the cluster?  I know the PR description does a good job of explaining this, but a comment here would be helpful.

Reply:
If I was a better person I would resend it with that change right now.

I am not a better person 😄 

(if the latest changes fail CI, I'll resend with the above recommendation as well)